webpack.config.js

```diff
@@ -31,7 +31,6 @@ module.exports = {
       },
       {
         test: /\.s?css$/i,
-        exclude: /(node_modules)/,
```

Comment:
correct me if i'm wrong, if you remove `exclude`, webpack will come to node_modules and parse everything inside? is it something we would like to do to all modules? maybe we should make ui-kit as exception and but keep node_modules excluded? 

cc. @fahad19